week_1/prob_1b.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
+
+cat web-server.log | awk -F " " '{print $9}' | sort | uniq -c | awk '{print $2 ": " $1}'
```

Comment:
can you print it's output in a file and push that file?

Reply:
Done